### PR TITLE
Change generate function to use buffered HTML generator

### PR DESCRIPTION
**Pull request by DarkWiiPlayer**

### CODE REVIEW

File: tableview.moon [hunk 1]
```diff
@@ -1,11 +1,14 @@
 mxml    = require "moonxml"
 loadkit = require "loadkit"
 inspect = require "inspect"
+html = mxml.html\buffered!
```
Comment (daelvn) [inline tableview.moon]: This is not aligned. You should know me better than that.

File: tableview.moon [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 
 moonxmlFind = loadkit.make_loader "lua"
 
 generate = (tbl) ->
-  template = mxml.html\loadluafile moonxmlFind "tableview.template"
-  return template tbl, inspect
+  html.buffer\truncate(0)
```
Comment (daelvn) [inline tableview.moon]: Remove parentheses here (it is not consistent with code style).

File: tableview.moon [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
+  template = html\loadluafile moonxmlFind "tableview.template"
+  template tbl, inspect
+  html.buffer\concat
```
Comment (daelvn) [inline tableview.moon]: Isn't a `!` missing here?

**Review by daelvn — CHANGES_REQUESTED** [review]

File: tableview.moon (hunk 3 above)
Comment (DarkWiiPlayer) [reply tableview.moon]: FFS this is why I hate switching between ruby and moonscript

**Review by daelvn — APPROVED** [review]

**Pull request merged**